common/Storyboarding/OsbSprite.cs

```diff
@@ -198,17 +200,17 @@ private void addCommand(ICommand command)
 
         private List<KeyValuePair<Predicate<ICommand>, IAnimatedValueBuilder>> displayValueBuilders = new List<KeyValuePair<Predicate<ICommand>, IAnimatedValueBuilder>>();
 
-        private AnimatedValue<CommandPosition> moveTimeline = new AnimatedValue<CommandPosition>();
```

Comment:
Not needed anymore?

Reply:
Not sure if that is the best way to get that info into the writer but I had changed these from private to protected so OsbAnimation can use them to construct the writer.

Reply:
reverted this change and added a factory for constructing the writer